src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -637,6 +641,32 @@ function validate_whitelist_ips(ips_to_validate) {
     }
 }
 
+///////////////////////////////////
+////  CONNECTION VALIDATIONS   ////
+///////////////////////////////////
+
+function validate_connection_args(user_input, action) {
+
+    //name is mandatory for all except LIST
+    if (action !== ACTIONS.LIST && !user_input.name) {
+        throw_cli_error(ManageCLIError.MissingCliParam, "CLI parameter 'name' is mandatory.");
```

Comment:
I think you need to edit the message of `ManageCLIError.MissingCliParam` you created and pass it to the `detail` argument information that is specific for this case.